.circleci/config.yml

```diff
@@ -96,7 +96,7 @@ aliases:
      name: upload coverage report
      command: |
        . /srv/zulip-py3-venv/bin/activate
-       pip install codecov && codecov \
+       pip install codecov==2.0.15 && codecov \
```

Comment:
s/undeterministic/nondeterminstic/ (the "un" version isn't a word AFAIK)

Reply:
Also can you add more details about the "gov find error"?

Reply:
(Like, are newer versions buggy, or is something else going on?)

Reply:
I updated the commit message with what I could find/know about it.

Reply:
I'd put the issue link in a comment, so it's handy when one's looking at this code.